Corrected reference to cache storage account location

paulkel opened the pull request:
Cache storage account was described as residing in target location. Corrected this to refer to source location.

azurecla commented:
Hi __@AzureDocsBot__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

jodoglevy commented:
Please ignore - testing for HackADoc.

paulkel committed:
Corrected reference to cache storage account location

Cache storage account was described as residing in target location. Corrected this to refer to source location.
